docs/mxdoc.py

```diff
@@ -87,10 +87,10 @@ def generate_doxygen(app):
 def build_mxnet(app):
     """Build mxnet .so lib"""
     if not os.path.exists(os.path.join(app.builder.srcdir, '..', 'config.mk')):
-        _run_cmd("cd %s/.. && cp make/config.mk config.mk && make -j$(nproc) DEBUG=1 USE_MKLDNN=0" %
+        _run_cmd("cd %s/.. && cp make/config.mk config.mk && make -j$(nproc) DEBUG=1 USE_MKLDNN=0 USE_CPP_PACKAGE=1" %
                 app.builder.srcdir)
     else:
-        _run_cmd("cd %s/.. && make -j$(nproc) DEBUG=1 USE_MKLDNN=0" %
+        _run_cmd("cd %s/.. && make -j$(nproc) DEBUG=1 USE_MKLDNN=0 USE_CPP_PACKAGE=1" %
```

Comment:
Do we need to build with DEBUG=1 if we're only interested in documentation?

Reply:
It's always been there. Not sure of the impact of removing it. Maybe I can test it out and remove it in another PR.

Reply:
That'd be great :)

Reply:
I remember that I added the DEBUG flag quite some time ago, but that was to build with debug symbols. Probably some copy&paste leftover. It doubles the buildtime though if I remember correctly